mmv1/products/apigee/api.yaml

```diff
@@ -660,3 +660,52 @@ objects:
         'Provisioning NAT IPs':
           'https://cloud.google.com/apigee/docs/api-platform/security/nat-provisioning'
       api: 'https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.instances.natAddresses'
+  - !ruby/object:Api::Resource
+    name: 'SyncAuthorization'
+    self_link: 'organizations/{{name}}:getSyncAuthorization'
+    base_url: ''
+    create_url: 'organizations/{{name}}:setSyncAuthorization'
+    update_url: 'organizations/{{name}}:setSyncAuthorization'
+    read_verb: :POST
+    update_verb: :POST
+    description: |
+      Authorize the Synchronizer to download environment data from the control plane.
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        required: true
+        description: |
+          Name of the Apigee organization.
+        url_param_only: true
+        input: true
+    properties:
+      - !ruby/object:Api::Type::Array
+        name: 'identities'
+        required: true
+        send_empty_value: true
+        description: |
+          Array of service accounts to grant access to control plane resources, each specified using the following format: `serviceAccount:service-account-name`.
+
+          The `service-account-name` is formatted like an email address. For example: my-synchronizer-manager-serviceAccount@my_project_id.iam.gserviceaccount.com
+
+          You might specify multiple service accounts, for example, if you have multiple environments and wish to assign a unique service account to each one.
+
+          The service accounts must have **Apigee Synchronizer Manager** role. See also [Create service accounts](https://cloud.google.com/apigee/docs/hybrid/v1.8/sa-about#create-the-service-accounts).
+        item_type: Api::Type::String
+      - !ruby/object:Api::Type::Fingerprint
+        name: 'etag'
+        description: |
+          Entity tag (ETag) used for optimistic concurrency control as a way to help prevent simultaneous updates from overwriting each other.
+
+          For example, when you call [getSyncAuthorization](https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations/getSyncAuthorization) an ETag is returned in the response.
+          Pass that ETag when calling the [setSyncAuthorization](https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations/setSyncAuthorization) to ensure that you are updating the correct version.
+          If you don't pass the ETag in the call to **setSyncAuthorization**, then the existing authorization is overwritten indiscriminately.
+
+          **Note**: We strongly recommend that you use the ETag in the read-modify-write cycle to avoid race conditions.
+
+          A base64-encoded string.
```

Comment:
If omitted, the etag will not be passed in the call. Then the existing resource will always be overwritten by the update. Is it a concern?

With `!ruby/object:Api::Type::Fingerprint`, the field etag is [computed](https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-6912-old..auto-pr-6912#diff-72d50d8fa04bb8ac299324ce37a8d165f342b9361c620c7e7753a836daeddf69R66).  It can avoid collision. 

The description for the etag is confused. I need to modify the description to make it match with `computed`.

Reply:
acc, understood.